adafruit_ads1x15/ads1015.py

```diff
@@ -67,6 +67,12 @@ def rate_config(self) -> Dict[int, int]:
     def _data_rate_default(self) -> Literal[1600]:
         return 1600
 
+    def _comp_low_thres_default(self) -> Literal[0x8000]:
+        return 0x8000
+
+    def _comp_high_thres_default(self) -> Literal[0x7FF0]:
+        return 0x7FF0
+
     def _conversion_value(self, raw_adc: int) -> int:
         value = struct.unpack(">h", raw_adc.to_bytes(2, "big"))[0]
-        return value >> 4
+        return value
```

Comment:
Don't change this. You'll change how `read()` works otherwise. Only `.value` needs to be the 16bit standard.

Reply:
In the ads1015 flow, I believe there are 2 unnecessary bit shifts. This is the right bit shift, which is immediately cancelled out by the left bit shift in the value property of analog_in.py. I removed both of these. There is no need to take in a 16-bit number, convert to 12, convert back to 16, and then report as 16. Removing both keeps .value at 16-bits for both ads1015 and ads1115.